### PR TITLE
EXAMPLES: Update orbit and trackball controls

### PR DESCRIPTION
**Pull request by looeee**
Update the orbit and trackball control examples to use the same scene as the fly controls. 
Also removed the postprocessing as I don't think it added that much and made the code more complicated for newcomers to read. 

Orbit controls: 
[before](https://threejs.org/examples/?q=contr#misc_controls_orbit) 
[after](https://rawgit.com/looeee/three.js/update_orbit_trackball_controls/examples/misc_controls_orbit.html)

Trackball controls:  
[before](https://threejs.org/examples/?q=contr#misc_controls_trackball) 
[after](https://rawgit.com/looeee/three.js/update_orbit_trackball_controls/examples/misc_controls_trackball.html)

[Current fly controls example](https://threejs.org/examples/?q=contr#misc_controls_fly) (with postprocessing)

### CODE REVIEW

**Comment by WestLangley** [comment]
Sorry, this does not make a whole lot of sense to me. `OrbitControls` is designed to be used when it is important for the camera to remain right-side-up. For example, when there is a "ground".

**Comment by looeee** [comment]
There is no ground in the current example either - the axis being preserved in this case is the Earth's north south axis, which I would say is just as valid as the axis of a bunch of triangles pointing upwards.

**Comment by WestLangley** [comment]
The current example could be improved, but it clearly demonstrates how to render a static scene without an animation loop.

File: examples/misc_controls_orbit.html [hunk 1]
```diff
@@ -48,101 +54,215 @@
...
+			var d, dPlanet, dMoon, dMoonVec = new THREE.Vector3();
+
+			var clock = new THREE.Clock();
 
 			init();
-			render(); // remove when using next line for animation loop (requestAnimationFrame)
-			//animate();
+
+			// render(); // add this when using controls.addEventListener( 'change', render ) for animation loop
+			animate(); // remove when using event listener for animation loop
```
Comment (WestLangley) [inline examples/misc_controls_orbit.html]: How does one use an event listener for an animation loop?

Comment (looeee) [reply examples/misc_controls_orbit.html]: That is a good point - I was just adapting the previous comment in the OrbitControls file. I'll change it to be clearer.

**Comment by looeee** [comment]
> it clearly demonstrates how to render a static scene without an animation loop.

That is not the purpose of this example though. It's intended for demonstrating how the controls work and I think that having the same scene for each of these 3 control types is very useful for that - even if it's not this particular scene.

**Comment by WestLangley** [comment]
>That is not the purpose of this example though. 

The current `OrbitControls` example was written specifically to show how to use it in a static scene without an animation loop.

The same technique does not work with `TrackballControls`.

**Comment by looeee** [comment]
OK, well if we do switch to this scene then it's still easy to use the event listener for rendering, we'd just have to forgo the planet's rotation.

**Comment by mrdoob** [comment]
I think I prefer the simplicity of the current examples.

File: examples/misc_controls_orbit.html [hunk 2]
```diff
@@ -48,101 +54,215 @@
...
 			function render() {
 
+				// rotate the planet and clouds
+
+				var delta = clock.getDelta();
+
+				meshPlanet.rotation.y += rotationSpeed * delta;
+				meshClouds.rotation.y += 1.25 * rotationSpeed * delta;
+
+				controls.update( delta );
```
Comment (WestLangley) [inline examples/misc_controls_orbit.html]: `OrbitControls.update()` does not have an argument.

File: examples/misc_controls_trackball.html [hunk 3]
```diff
@@ -49,19 +53,46 @@
...
+
+			var d, dPlanet, dMoon, dMoonVec = new THREE.Vector3();
+
+			var clock = new THREE.Clock();
 
 			var cross;
 
 			init();
-			animate();
+			// render(); // add this when using controls.addEventListener( 'change', render ) for animation loop
```
Comment (WestLangley) [inline examples/misc_controls_trackball.html]: Have you ever been able to use `TrackballControls` without an animation loop?

**Comment by looeee** [comment]
OK, seems like you guys prefer the current examples. I'm fine with that so I'll close this.

**Pull request closed without merging**